pom.xml

```diff
@@ -128,8 +128,8 @@
       <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
       <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
-      <maven.compiler.source>1.7</maven.compiler.source>
-      <maven.compiler.target>1.7</maven.compiler.target>
+      <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
@cshannon @johnament what about we do it now.. and listen who complains about it? :)

If we get many complains we can always revert and redo a next release.. such as 1.5.1 or 1.6.0.